integrations-and-supported-tools/colab/file.txt

```diff
@@ -1 +0,0 @@
-Welcome to Neptune
```

Comment:
@SiddhantSadangi I don't think this file should be here

Reply:
yeah, this was deleted in this PR